src/components/Modals/SaveCourseModal.vue

```diff
@@ -162,6 +162,11 @@ export default defineComponent({
         collection => !this.checkedCollections.includes(collection)
       );
 
+      // If no specific collections were selected, and no prevoius selections were removed, add to 'All'
+      if (addedToCollections.length === 0 && deletedFromCollections.length === 0) {
+        console.log('added to All');
```

Comment:
Just to clarify the functionality: 
anytime you save _any_ course, it should end up in the "all" collection, whether you add it to another specific collection or not?

Reply:
Also, on a completely separate note, should the other drop downs for collections close if you open a different one? That is, is it possible for all of the collection drop downs to be open at the same time?